MAINTENANCE.md

```diff
@@ -77,8 +78,9 @@ There are 9 configuration files located in `.github/workflows`:
 
    This is run on every commit to the *master* and Pull Request branches.
    It is also scheduled to run daily on the *master* branch.
-   In draft Pull Requests, only one job (Linux + Python latest)
-   is triggered to save on Continuous Integration resources.
+   In draft Pull Requests, only two jobs on Linux (Python 3.7 / NumPy 1.17 and
+   Python 3.9/ NumPy 1.20) are triggered to save on Continuous Integration
+   resources.
```

Comment:
Do you really want to hardcode the version numbers here? It means we have to remember to update this file when we drop support for any old versions. Perhaps we can rephrase it like

"minimum Python / NumPy versions and latest Python / NumPy versions)"

Reply:
Done in 1016331f6ae94ec7ac744588dd2b64fdf685a77a.